tests/Unit/ControlSystem/Test_FunctionOfTime.cpp

```diff
@@ -0,0 +1,133 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "ControlSystem/FunctionOfTime.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+SPECTRE_TEST_CASE("Unit.ControlSystem.FunctionOfTime.PiecewisePolynomial",
+                  "[ControlSystem][Unit]") {
+  double t = 0.0;
+  const double dt = 0.6;
+  const double final_time = 4.0;
+  const constexpr size_t deriv_order = 3;
+  using value_type = std::array<DataVector, deriv_order + 1>;
```

Comment:
You only use this type alias once, just write the type there.

Apply to other tests as well